test/library/packages/OrderedSet/general/add/EXECOPTS

```diff
@@ -0,0 +1 @@
+--memLeaks
```

Comment:
It is only partially true today: We do multilocale and release/examples testing with memleaks by default. But full testing is still as it has been since forever, and we don't throw in memleaks by default. So this would make this whole folder getting tested with memory leaks and I don't see anything wrong with it. If anything it'll make the testing for this folder stronger.

In the future, we hope that it'll change and we'll use `memLeaks` by default at least in some configurations of our nightly testing, and then, this file will be redundant.